langroid/vector_store/qdrantdb.py

```diff
@@ -220,6 +246,28 @@ def create_collection(self, collection_name: str, replace: bool = False) -> None
             logger.setLevel(logging.INFO)
             logger.info(collection_info)
             logger.setLevel(level)
+    
+    def get_sparse_embeddings(self, inputs: List[str]) -> List[SparseVector]:
+        if not self.config.use_sparse_embeddings: return []
+        import torch
+        tokens = self.sparse_tokenizer(inputs, return_tensors='pt', truncation=True, padding=True)
+        output = self.sparse_model(**tokens)
+        vectors = torch.max(
```

Comment:
formatting is missing indents. Make sure to run `make lint` to fix style issues.